templates/add_blurb.html

```diff
@@ -18,10 +18,10 @@ <h3>📜🤖 Blurb it again?</h3>
                 <input type="hidden" name="csrf" value="{{ csrf }}">
                 <div class="mb-3">
                     <div class="col form-inline">
-                        <label for="bpo_number" class="form-inline">bpo-</label><input type="text" class="form-control form-inline" placeholder="12345" id="bpo_number" name="bpo_number" required>
+                        <label for="issue_number" class="form-inline">gh-</label><input type="text" class="form-control form-inline" placeholder="12345" id="issue_number" name="issue_number" required>
                     </div>
                     <div class="col form-inline">
-                        <small id="bpo-help" class="form-text text-muted">The bugs.python.org issue number.</small>
+                        <small id="issue_number-help" class="form-text text-muted">The GitHub issue number.</small>
```

Comment:
For consistency with the description below, either `CPython` should be added here, or removed below. It might also be useful to add some emphasis on `issue` number and `pull request` to prevent people from mixing them up.